NuGet.config

```diff
@@ -32,34 +22,14 @@
     <packageSource key="dotnet-tools">
       <package pattern="*" />
     </packageSource>
-    <packageSource key="dotnet8">
-      <package pattern="*" />
-    </packageSource>
-    <packageSource key="dotnet8-transport">
-      <package pattern="*" />
-    </packageSource>
     <packageSource key="dotnet9">
       <package pattern="*" />
     </packageSource>
     <packageSource key="dotnet9-transport">
       <package pattern="*" />
     </packageSource>
-    <packageSource key="dotnet10">
-      <package pattern="*" />
-    </packageSource>
-    <packageSource key="dotnet10-transport">
-      <package pattern="*" />
-    </packageSource>
     <packageSource key="richnav">
       <package pattern="*" />
     </packageSource>
   </packageSourceMapping>
-  <disabledPackageSources>
```

Comment:
I think it should be better to keep the section, and just leave it empty as it is. That way, when we flow internal dependencies in preparation for our next release, maestro can still add sources bellow in the section that they should go.

Reply:
Maestro can add this section automatically, I believe